client/webserver/site/src/js/markets.js

```diff
@@ -188,14 +191,60 @@ export default class MarketsPage extends BasePage {
     ws.registerRoute(epochOrderRoute, data => { this.handleEpochOrderRoute(data) })
     // Bind the wallet unlock form.
     bindOpenWallet(app, page.openForm, async () => { this.openFunc() })
-    // Create a wallet
+    // Create a wallet.
```

Comment:
This is not a sentence so doesn't get a full stop, nor is it relevant to this PR.